src/algorithms/EpsilonConstraint.jl

```diff
@@ -104,9 +104,7 @@ function optimize_multiobjective!(
         MOI.GreaterThan{Float64}, left, 1.0
     end
     ci = MOI.add_constraint(model, f1, SetType(bound))
-    # Set a finite upper bound on the number of iterations so that we don't loop
-    # forever.
-    for i in 1:ceil(Int, abs(right - left) / ε + 3)
+    while true
```

Comment:
Is there any harm in leaving the explicit for-loop?

Reply:
It might stay on the same solution once and miss the "last" one if the for-loop ends prematurely. I think taking the max/min ensures it will stop